packages/driver/src/multi-domain/errors.ts

```diff
@@ -0,0 +1,31 @@
+import type { $Cy } from '../cypress/cy'
+import $errUtils, { ErrorFromProjectRejectionEvent } from '../cypress/error_utils'
+
+export const handleErrorEvent = (cy: $Cy) => {
+  return (handlerType: string, frameType?: string) => {
```

Comment:
Are there other types for` frameType` besides `spec` or `app` on errors?
```suggestion
  return (handlerType: string, frameType?: 'spec' | 'app') => {
```

Reply:
I believe those are the only two types. Updated 1dd8269